src/main/java/task/Task.java

```diff
@@ -0,0 +1,32 @@
+package task;
+
+public class Task {
```

Comment:
Maybe we can make this abstract? Since we won't really instantiate this class directly if i'm not wrong